cluster/definition.go

```diff
@@ -41,6 +41,15 @@ type NodeIdx struct {
 	ShareIdx int
 }
 
+// WithV1x4 provides an option to create a new definition with v1.4 including a creator.
+// TODO(corver): Remove this and add creator to NewDefinition when releasing v1.4.
```

Comment:
can we have a ticket for this TODO with label as release version, to better track the tickets to be handled before next release?

Reply:
sure